CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+
+**Order related GraphQL mutations**
+A `createRefund` GraphQL mutation has been added to allow refunds to be processed via GraphQL instead of older meteor methods.
+
+**Dataloaders introduced into the codebase**
+Added the first couple of DataLoaders for Shops and SimpleInventory. Shops will benefit product listing pages as product and variant resolvers all query Shops collection for each product. SimpleInventory query benefits from batching.
+
+**Bulk simple inventory mutation**
+When running large inventory import via an external sync system, the lack of a bulk mutation made the import takes many hours. This update introduces a bulk import option which cuts down import time significantly.
+
+**Overhaul of `eslint` rules, and bulk fix of over 500 `eslint` warnings**
+We've updated to using v2.0.0 of our [Reaction ESLint Config](https://github.com/reactioncommerce/reaction-eslint-config) package, and have brought our `eslint` warnings down from 800 to under 300.
+
+## Feature
+
+- feat: catalog partial publish / rerun transformations ([#5355](https://github.com/reactioncommerce/reaction/pull/5355))
+- feat(product table): show Action dropdown, even if no products are selected ([#5391](https://github.com/reactioncommerce/reaction/pull/5391))
+- feat: filter products by id in products table ([#5387](https://github.com/reactioncommerce/reaction/pull/5387))
+- feat: editable tag slug ([#5382](https://github.com/reactioncommerce/reaction/pull/5382))
+- feat: add createRefund GraphQL mutation ([#5354](https://github.com/reactioncommerce/reaction/pull/5354))
+- feat: add handle to tag autosuggest results ([#5262](https://github.com/reactioncommerce/reaction/pull/5262))
+- featL Add DataLoaders for Shops and SimpleInventory ([#5294](https://github.com/reactioncommerce/reaction/pull/5294))
+- feat: Introduce DataLoaders infrastructure ([#5280](https://github.com/reactioncommerce/reaction/pull/5280))
+- feat: new button component to wrap MUI button with isWaiting loading state ([#5266](https://github.com/reactioncommerce/reaction/pull/5266))
+- feat: bulk simple inventory mutation ([#5229](https://github.com/reactioncommerce/reaction/pull/5229))
+- feat: integrate catalyst ConfirmDialog component from new Catalyst package ([#5372](https://github.com/reactioncommerce/reaction/pull/5372))
+
+## Fixes
+
+- fix: set slugified title as permalink ([#5264](https://github.com/reactioncommerce/reaction/pull/5264))
+- fix Add migration for index ([#5358](https://github.com/reactioncommerce/reaction/pull/5358))
+- fix: red textfields ([#5371](https://github.com/reactioncommerce/reaction/pull/5371))
+- fix: Make catalog indexes unique for product ([#5350](https://github.com/reactioncommerce/reaction/pull/5350))
+- fix: fix consistent-return eslint warnings ([#5334](https://github.com/reactioncommerce/reaction/pull/5334))
+- fix(5305-slow-simplschema-validate): replace simpl-schema.validate() … ([#5306](https://github.com/reactioncommerce/reaction/pull/5306))
```

Comment:
I'd favor removing the issue reference found in the parens here unless you think it adds value